prbt_hardware_support/README.md

```diff
@@ -16,6 +16,8 @@ The following table shows the names and the possible argument values of the feat
 
 With `pg70` referring to __Schunk PG plus 70__. For more on gripper models see [prbt_grippers](https://github.com/PilzDE/prbt_grippers).
 
+The optional argument `iso10218_support` can be used to start ROS without connecting to a safety controller. Keep in mind that this disables all safety features, so it should only be used for debugging purposes.
```

Comment:
Does it make sense to document this argument here, if it doesn't even occur in this package?

Reply:
Good question. We chose to describe the supported hardware configurations/features in the readme of prbt_hardware_support. That is why the arguments for `robot.launch` appear here. So I would keep it for now.
When we come up with a better package structure in the future, it should get more consistent.

Reply:
But I guess the whole section could be moved to the `prbt_support` documentation as well.

Reply:
I would suggest to move the section `How-to activate and deactivate hardware features` to prbt_support/README, since it describes the arguments of `robot.launch`. We could then add a reference from prbt_support/README to prbt_hardware_support/README for more information about the Safety Features.
Alternatively we duplicate the section `How-to activate and deactivate hardware features`, which was your suggestion, correct?
@ct2034 Do you have any additional input?